<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HogMainWin</class>
 <widget class="QMainWindow" name="HogMainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HogApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="videoHLayout">
      <item>
       <widget class="VideoWidget" name="videoWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="HogWidget" name="hogWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlsHLayout">
      <item>
       <widget class="QPushButton" name="playPauseBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Paused</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+P</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="openDirBtn">
        <property name="text">
         <string>OpenDirectory</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+D</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HogWidget</class>
   <extends>QWidget</extends>
   <header>hogwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
